db/migrate/20200703192525_add_notes_and_distribution_limit_to_partners.rb

```diff
@@ -0,0 +1,6 @@
+class AddNotesAndDistributionLimitToPartners < ActiveRecord::Migration[6.0]
+  def change
+    add_column :partners, :note, :text
+    add_column :partners, :distribution_limit, :integer
```

Comment:
I'm not sure you need two migrations. You can probably change this one to:
`add_column :partners, :distribution_limit, :integer, default: 0`

Reply:
I'd also add `null: false` option here. Since I don't think we'll want to see `distribution_limit=nil`